### PR TITLE
Turn off http errors when fetching Microsoft avatars

### PR DESCRIPTION
**Pull request by elite123**
Fixes https://github.com/SocialiteProviders/Providers/issues/824 where a 404 error bubbles up if the avatar doesn't exist

### CODE REVIEW

**Review by atymic — COMMENTED** [review]
Should we not use a try/catch and return null?

**Comment by elite123** [comment]
@atymic I have updated to catch \GuzzleHttp\Exception\ClientException

File: src/Microsoft/MicrosoftUser.php [hunk 1]
```diff
@@ -14,16 +14,21 @@ class MicrosoftUser extends User
...
+                [
+                    RequestOptions::HEADERS => [
+                        'Accept'        => 'image/*',
+                        'Authorization' => 'Bearer '.$this->token,
+                    ],
+                ]
+            );
+
+            return (new MicrosoftAvatar())->setResponse($response);
+        } catch (\GuzzleHttp\Exception\ClientException $e) {
```
Comment (lucasmichot) [inline src/Microsoft/MicrosoftUser.php:30]: Can you import this `ClientException` ?

**Review by lucasmichot — CHANGES_REQUESTED** [review]

File: src/Microsoft/MicrosoftUser.php (hunk 1 above)
Comment (elite123) [reply src/Microsoft/MicrosoftUser.php]: @lucasmichot Updated to import `ClientException`

**Review by atymic — APPROVED** [review]

**Review by atymic — APPROVED** [review]

**Pull request merged**